packages/map/src/CdcMapComponent.styles.css

```diff
@@ -0,0 +1,9 @@
+.map-container {
```

Comment:
We reserve capital file naming convention for Components only.

this file should be cdcMapComponent.styles.css